docs/performance/app_open_performance.md

```diff
@@ -55,24 +55,23 @@ To further understand why app open times are slow it is important to find out ex
 
 The engine goes through many stages when a user wants to attach to an app, and some of those stages will depend on whether or not the app is already loaded into memory. The following progress report shows a typical case of a "cold open", where the app has not already been loaded into memory:
 
-![Progress_report_cold_open.png](images/Progress_report_cold_open.png)
+![Progress_report_cold_open.png](images/Progress_report_cold_open_highlighted.png)
 
 Notice how much of the time (about 48 seconds in total) is spent on loading fields and tables. This is the engine reading the QVF, unpacking the data, and building the data structures required to perform calculations on the app. Compare that to the following progress report for the exact same app:
 
 ![Progress_report_warm_open.png](images/Progress_report_warm_open.png)
 
 Here we can see that the loading of fields and tables are not present. In fact, if we compare line four of the two progress reports, we can see that the first one states "Loading from storage" while the second one reads "Attached to loaded document". This means that the app was alreay loaded into memory when the second connection was performed.
 
-If users complain that apps are sometimes slow to open, then this is the first thing to look for. If it is clear that the loading of fields and tables is the main cause of slow app open times, then implementing some form of pre-caching technique in your environment might be a good option.
```

Comment:
Can we refer to another whitepaper for pre-caching or pre-loading of apps?
Or perhaps refer it to qs-admin-playbook page in future

Reply:
I do have a cross reference to the "Cache warming" section of the playbook in there. Is that sufficient, or are you referring to something else?

Reply:
The cross reference is good enough. Thanks